src/modules/makeTry.js

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable no-console */
+'use strict';
+
+const readline = require('readline');
+const { getHint } = require('./getHint');
+
+function makeTry(winCombination) {
+  const terminal = readline.createInterface({
+    input: process.stdin,
+    output: process.stdout,
+  });
+
+  terminal.question('Enter your 4-digits number: ', (number) => {
+    if (number === winCombination) {
+      console.log('Nice! You guessed it!');
+      terminal.close();
+    } else if (number.length !== 4) {
+      console.log('This should be a 4-digits number! Try again.');
+      terminal.close();
```

Comment:
Instead of closing the terminal and recalling 'makeTry(winCombination)', consider using a loop to continuously prompt the user for input until a valid input is received.